cogs/user.py

```diff
@@ -237,7 +263,9 @@ async def vc(self, ctx, *usernames):
 
         query = Query()
         if usernames == []:
-            usernames = [query.get_handle(ctx.author.id, ctx.guild.id)]
+            username = query.get_handle(ctx.author.id, ctx.guild.id)
+            if username:
+                usernames = [username]
 
         users = await asyncio.gather(*[query.get_user(username)
                                      for username in usernames])
```

Comment:
I didn't write this command lol

Reply:
wait I think what happens is it's just an empty list, so it doesn't filter any contests

Reply:
```python
    username=query.get_handle(ctx.author.id, ctx.guild.id)
            if username:
                usernames = [username]
```
If no handle is found, `get_handle` returns None, after that it would continue down as None, not a empty list